Doc/library/dataclasses.rst

```diff
@@ -51,24 +51,24 @@ Module contents
    This function is a :term:`decorator` that is used to add generated
    :term:`special methods <special method>` to classes, as described below.
 
-   The ``@dataclass`` decorator examines the class to find
+   The :func:`!@dataclass` decorator examines the class to find
```

Comment:
This will include the parentheses:

> The `@dataclass()` decorator ...

To skip them, do we need something like this?

```suggestion
   The :func:`!@dataclass <dataclass>` decorator examines the class to find
```

Reply:
Ah, yes. That's unfortunate. I'll look into it later.

Reply:
I think it is too verbose to use such workaround for all the `@dataclass` mentions in this file. I suggest we leave those change out of this PR for now. (Looking forward to the `:deco:` role 😎)

Reply:
Sure, so do you mean you'll revert them to the original ``` :func:`!@dataclass` ``` etc. or want to keep the changed ``` `@dataclass <!dataclass>` ```?

Reply:
I meant revert them to how they are in `main` right now; IOW just use double backticks.

Reply:
Hm, OTOH, the verbose style is widely used in this file. It might be better to align with that style.

Reply:
Addressed in 163aaa5d5e3efb6d6c5eb38eb6b07ab9cf7c418a